lib/ast/node.rb

```diff
@@ -211,7 +211,33 @@ def to_sexp(indent=0)
 
       sexp
     end
-    alias :inspect :to_sexp
```

Comment:
to_s is defined, so i did't want to touch that
figured you can easily do that line if you want it. That change hasn't really got anything to do with the inspect changing.